### ISSUE TITLE
[Fixed in GH-6052] No visual focus feedback for menu items on desktop

### ISSUE THREAD

**Issue by Zeyber** [SAP/spartacus#4148]
![image](https://user-images.githubusercontent.com/11735817/63436491-136b1980-c429-11e9-8a96-5264acf7ff3a.png)

**Comment by KateChuen** [comment]
Chrome:
Visual focus and keyboard accessibility on menu items OK. But the border is not properly displayed on the first one:
![Screen Shot 2019-10-21 at 3 11 18 PM](https://user-images.githubusercontent.com/45409515/67235925-1fb62780-f416-11e9-84b2-e2dc6b704f64.png)


Safari & Firefox:
Missing a lot of visual focus and keyboard accessibility on menu items.
<img width="919" alt="Screen Shot 2019-10-21 at 3 11 51 PM" src="https://user-images.githubusercontent.com/45409515/67235935-26dd3580-f416-11e9-84ff-d7df9fc8887a.png">

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.

![Screen Shot 2020-01-30 at 2 13 20 PM](https://user-images.githubusercontent.com/15113219/73482148-305a3000-436b-11ea-9f25-a311b20e5beb.png)

![Screen Shot 2020-01-30 at 2 14 07 PM](https://user-images.githubusercontent.com/15113219/73482149-305a3000-436b-11ea-8a00-bcf06bda1fc6.png)

![Screen Shot 2020-01-30 at 2 15 11 PM](https://user-images.githubusercontent.com/15113219/73482154-3223f380-436b-11ea-9146-d3dc84381179.png)

**Comment by giancorderoortiz** [comment]
Chrome: ok.  👍 
Firefox v71.0 (Unable to navigate to subcategories (via tab) in main-navigation and confirm focus. 👎 
Safari: pending.

**Comment by kacperknapik** [comment]
Tested in Firefox: 👍 
Safari: **???**

**Comment by KateChuen** [comment]
Tested on dev3, on safari.

Ok now. Just a question, is it ok to have focus on an element that we can't navigate to? For eg, the CAMERAS subcategory. @developpeurweb can you confirm please?

<img width="1173" alt="Screen Shot 2020-03-26 at 9 39 31 AM" src="https://user-images.githubusercontent.com/45409515/77653482-20636480-6f46-11ea-90f6-d0f749c14337.png">

**Comment by developpeurweb** [comment]
@KateChuen 
I'm unable to reproduce this with latest `develop + dev-com-17` as shown in the animation below.

![2020-04-01_09-53-11 (1)](https://user-images.githubusercontent.com/15113219/78154268-117b2700-740a-11ea-8b9a-e8995b530845.gif)

but still, at least for this first accessibility milestone this OK and wouldn't be an issue, since **navigation will have to be refactored for the 2nd accessibility milestone** aimed at Screen Readers GH-7120.

Then we will have to decide how to handle this. Right now, by focusing on this H5 is the only way a Screen Reader will read it when focusing and announce it as having items in the context of the menu, but the ideal scenario, off the top of my head, would be to either: 

1. having a combination of ARIA attributes with a list, or 
2. having the subcategory being an actual link leading to a filtered result

**Comment by KateChuen** [comment]
yes confirmed, it's no longer reproducible with the latest develop. Was fixed in #6795.
Verified on chrome, firefox & safari.
